ui/app/templates/allocations/allocation/task/fs.hbs

```diff
@@ -1 +1,70 @@
-{{outlet}}
\ No newline at end of file
+{{task-subnav task=task}}
+<section class="section closer">
+  {{#if task.isRunning}}
+    {{! template-lint-disable table-groups }}
+    <table class="fs-explorer table is-compact">
+      <thead>
+        <tr>
+          <td colspan="3">
+            <nav class="breadcrumb fs" data-test-fs-breadcrumbs>
+              <ul>
+                <li class={{if breadcrumbs "" "is-active"}}>
+                  {{#link-to "allocations.allocation.task.fs-root" task.allocation task activeClass="is-active"}}
+                    {{task.name}}
+                  {{/link-to}}
+                </li>
+                {{#each breadcrumbs as |breadcrumb|}}
+                  <li class={{if breadcrumb.isLast "is-active"}}>
+                    {{#link-to "allocations.allocation.task.fs" task.allocation task breadcrumb.path activeClass="is-active"}}
+                      {{breadcrumb.name}}
+                    {{/link-to}}
+                  </li>
+                {{/each}}
+              </ul>
+            </nav>
+          </td>
+        </tr>
+      </thead>
+      {{#if isFile}}
+        <tbody>
+          <tr>
+            <td>
+              <div data-test-file-viewer>placeholder file viewer</div>
+            </td>
+          </tr>
+        </tbody>
+      {{else}}
+        {{#if directoryEntries}}
+          <thead>
+            <tr>
+              <th class="is-two-thirds">Name</th>
+              <th class="has-text-right">File Size</th>
+              <th class="has-text-right">Last Modified</th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#each (append directories files) as |entry|}}
+              {{fs-directory-entry path=path task=task entry=entry}}
+            {{/each}}
+          </tbody>
+        {{else}}
+          <tbody>
+            <tr data-test-entry>
+              <td colspan="3">
+                <span data-test-empty-icon>{{inline-svg "alert-circle-outline" class="icon"}}</span>
```

Comment:
Can this use the `x-icon` helper instead?

Reply:
oh yes, of course, thanks!